src/state/apis/swapper/helpers/validateQuoteRequest.ts

```diff
@@ -29,8 +29,8 @@ export const validateQuoteRequest = ({
     return [{ error: TradeQuoteRequestError.NoConnectedWallet }]
   }
 
-  const walletSupportsSellAssetChain = walletSupportedChainIds.includes(sellAsset.chainId)
-  const walletSupportsBuyAssetChain = walletSupportedChainIds.includes(buyAsset.chainId)
+  const walletSupportsSellAssetChain = walletConnectedChainIds.includes(sellAsset.chainId)
```

Comment:
Not a question but rather a note - looks like this one is mostly here for paranoia now (and already in develop): assuming we got the supported sell assets detection right, we'll never hit this with an unsupported sell asset.

Definitely makes sense to keep it though, just pointing I was unable to test this didn't break, since you can't get into a sell asset unsupported by wallet in the first place